actionpack/test/controller/log_subscriber_test.rb

```diff
@@ -92,6 +92,7 @@ class ACLogSubscriberTest < ActionController::TestCase
 
   def setup
     super
+    ActionView::Base.enable_fragment_cache_logging = true
```

Comment:
Reads a bit like a smell that we have to touch this config here as well as in the other tests. Perhaps there's a lower level of cache logging we can hook into?